### PR TITLE
#30536 PropertyAccessor->getValue disable exception

### PR DESCRIPTION
**Pull request by dimabory**
| Q             | A
| ------------- | ---
| Branch?       | master 
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes  
| Fixed tickets | #30536 
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/11156

### CODE REVIEW

File: src/Symfony/Component/PropertyAccess/PropertyAccessor.php [hunk 1]
```diff
@@ -70,10 +71,11 @@ class PropertyAccessor implements PropertyAccessorInterface
      * Should not be used by application code. Use
      * {@link PropertyAccess::createPropertyAccessor()} instead.
      */
-    public function __construct(bool $magicCall = false, bool $throwExceptionOnInvalidIndex = false, CacheItemPoolInterface $cacheItemPool = null)
+    public function __construct(bool $magicCall = false, bool $throwExceptionOnInvalidIndex = false, bool $throwExceptionOnInvalidPropertyPath = true, CacheItemPoolInterface $cacheItemPool = null)
```
Comment (nicolas-grekas) [inline src/Symfony/Component/PropertyAccess/PropertyAccessor.php]: that's a hard BC break, we don't accept them adding the argument last could work

Comment (dimabory) [reply src/Symfony/Component/PropertyAccess/PropertyAccessor.php]: Got it. Will change arguments  order.

**Comment by dimabory** [comment]
⬆ up to review

**Review by webmozart — APPROVED** [review]
Thanks for your contribution! Looks good to me :+1:

**Review by Simperfit — APPROVED** [review]

**Review by michaelcullum — APPROVED** [review]

**Comment by chalasr** [comment]
I just pushed a commit that updates FrameworkBundle+PropertyAccess CHANGELOG files.

**Review by chalasr — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @dimabory.

**Pull request merged**